Doc/library/datetime.rst

```diff
@@ -503,7 +502,7 @@ Other constructors, all class methods:
 .. classmethod:: date.fromordinal(ordinal)
 
    Return the date corresponding to the proleptic Gregorian ordinal, where
-   January 1 of year 1 has ordinal 1.
+   January 1 of year 1 has ordinal ``1``.
```

Comment:
I'm not sure about this one; IMO it's best kept unchanged.

```suggestion
   January 1 of year 1 has ordinal 1.
```

Reply:
Only literals should receive markup; I added this remark since I don't think it qualifies as a code literal.